src/logging.js

```diff
@@ -7,12 +7,12 @@ import stream from "stream";
 import expressWinston from "express-winston";
 import winston from "winston";
 
-import {LoggingWinston as StackdriverTransport} from "@google-cloud/logging-winston";
+import * as lw from "@google-cloud/logging-winston";
```

Comment:
nit: Would you be open to renaming "lw" to "loggingWinston" (or some such) to make its use a bit more obvious?

Reply:
Sure. I did this to be consistent with the graphql-gateway.